Essentials/src/messages_en.properties

```diff
@@ -613,6 +613,7 @@ warpList={0}
 warpListPermission=§4You do not have permission to list warps.
 warpNotExist=§4That warp does not exist.
 warpOverwrite=§4You cannot overwrite that warp.
+warpCreate=\u00a74You cannot create that warp.
```

Comment:
Don't add translations to files other than the default `messages.properties`, as [Crowdin](https://crowdin.com/project/essentials-2) will overwrite them. If you have translations to submit, it should be made there.

Reply:
Even better. Reverted.